Redesign project page

ubergesundheit opened the pull request:
This PR superseedes #77 

For now only as draft. I would appreciate your input!

k-nut commented:
@ubergesundheit if you rebase this on the latest ˋmainˋ now we should get the new Percy preview for your changes :)

k-nut commented:
@ubergesundheit I would personally leave the image as big as it currently is. Other than that I think this looks much better - and more usable - already!

ubergesundheit commented:
> I would personally leave the image as big as it currently is.

Yes good point. Reverted that change and also removed the lab logos at the start of the project page. They are repeated at the bottom anyway

Another change I did is to only render the `img` tag when the file is available on the project and lab pages.

ubergesundheit commented:
@okfde/codeforde-devs care to take another look? I‘ll resolve the git conflicts and use the simplified brackets @k-nut introduced. 

Any stylistic comments appreciated!

digital-codes commented:
Like the new project layout much better. Most project have just one labs, so maybe we change "Projekt-Mitglieder & Labs" to "Mitwirkende" or "Team". In case there would be more than one lab could we accommodate more than one Logo?

On Klimawatch, there is a <a href="https://twitter.com/krtfflslt">kartoffelsalat</a>, Münster instead of the link.
On Lorawan, there is a blonk missing between name and email: Jens Vorwergemail 
I didn't check all projects though.
In the project listing I'd prefer to active projects been listed before closed or archived ones.

I'd suggest to have the same new layout for the lab pages, there are still these strange collapsible items ...

The Ressources page is still broken: the resources are not centered but in the left column.
Also, I get a 404 on the "Handbuch" in the preview:
https://deploy-preview-216--hungry-heisenberg-4bd67f.netlify.app/ressourcen/ressourcen/Code-for-Germany_Handbuch-Krisenresilienz.pdf
In the blog list, I would remove the "cfg auf einen blick" headline in the image for "neuer wenauftritt" because it indicates a different topic (not the website work) 
I do NOT like the big blue monster biceps with the enormous donation box next to it. Can we make this smaller?

k-nut commented on themes/codefor-theme/layouts/projekte/single.html:
```diff
@@ -1,172 +1,74 @@
+            </li>
+            </ul>
+            <div class="mb-5">
+              <div class="project-labs-list">
```
I think the bootstrappy way here would be:
```suggestion
              <div class="d-flex justify-content-evenly">
```
I'm not a big fan of utility classes personally but at least this would be consistent with the rest of the code

ubergesundheit replied:
I'll also remove the class from `base.sass`

k-nut commented:
@ubergesundheit I added one comment regarding bootstrap classes.

Could you also please rebase/fix conflicts? :)

Overall, I quite like this!

ubergesundheit commented:
> Most project have just one lab, so maybe we change "Projekt-Mitglieder & Labs" to "Mitwirkende" or "Team". In case there would be more than one lab could we accommodate more than one Logo?

I like this! Changed the headline to `Mitwirkende`

> On Klimawatch, there is  "a href = https: //twitter.com/krtfflslt>kartoffelsalat</a, Münster" instead of the link.
> On Lorawan, there is a blonk missing between name and email: Jens Vorwergemail 

Fixed

> In the project listing I'd prefer to active projects been listed before closed or archived ones.

Could you create a separate issue for this?

> I'd suggest to have the same new layout for the lab pages, there are still these strange collapsible items ...
> The Ressources page is still broken: the resources are not centered but in the left column.
> Also, I get a 404 on the "Handbuch" in the preview:
> https://deploy-preview-216--hungry-heisenberg-4bd67f.netlify.app/ressourcen/ressourcen/Code-for-Germany_Handbuch-Krisenresilienz.pdf

This is because this branch was not up to date when you checked.

> In the blog list, I would remove the "cfg auf einen blick" headline in the image for "neuer wenauftritt" because it indicates a different topic (not the website work) 
I do NOT like the big blue monster biceps with the enormous donation box next to it. Can we make this smaller?

Could you create a separate issue for this?

k-nut reviewed (approved):
LGTM :shipit:
